README.md

```diff
@@ -1,167 +1,87 @@
+## Benefits 
 
-NVTabular is designed to support data scientists and machine learning (ML) engineers train (deep learning) recommender systems and resolve tabular data problems by allowing them to:
+Our ultimate goal is faster iteration on massive tabular datasets, both for experimentation during training, and also for production model responsiveness. NVTabular is designed to support data scientists and machine learning (ML) engineers train (deep learning) recommender systems and resolve tabular data problems by allowing them to:
 
-* prepare datasets quickly and easily for experimentation so that more models can be trained
+* prepare datasets quickly and easily for experimentation so that more models can be trained.
 * work with datasets that exceed GPU and CPU memory without having to worry about scale.
 * focus on what to do with the data, and not how to do it, using our abstraction at the operation level.
 
-It is also meant to help ML/Ops Engineers with deploying models into production by providing faster dataset transformation. This makes it easy for production models to be trained more frequently and kept up to date, helping improve responsiveness and model performance.
+NVTabular also helps ML/Ops engineers with deploying models into production by providing faster dataset transformation. This makes it easy for production models to be trained more frequently and kept up to date, helping improve responsiveness and model performance.
 
-The library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do per epoch shuffles orders of magnitude faster than a full shuffle of the dataset. Loading tabular data is almost always a training bottleneck. In our benchmarking, we've seen 10x improvements in training time on the GPU relative to the native dataloaders.
+## Core Features
 
-Extending beyond model training, we plan on integrating with model-serving frameworks like NVIDIA’s Triton Inference Server. This will create a clear path to production inference for these models and allow the feature engineering and preprocessing steps performed on the data during training to be automatically applied to incoming data during inference.
+NVTabular supports the following core features:
+* [Framework Interoperability](../docs/source/core_features.md#framework-interoperability)
+* [Multi-GPU Support](../docs/source/core_features.md#multi-gpu-support)
+* [Multi-Node Support](../docs/source/core_features.md#multi-node-support)
+* [Multi-Hot Encoding and Pre-existing Embeddings](../docs/source/core_features.md#multi-hot-encoding-and-pre-existing-embeddings)
+* [CPU Support](../docs/source/core_features.md#cpu-support)
```

Comment:
I like having this core features list!

Some thoughts on this:
* 'Framework Interoperability' isn't clear what frameworks we're talking about - we should mention something like 'TensorFlow, PyTorch and HugeCTR Interoperability' instead
* CPU Support is currently scheduled for 0.5 - and I don't think we should be highlighting as a core feature yet
* Inference ? Should it be here or in the benefits section?